source/mongo.txt

```diff
@@ -4,417 +4,26 @@ The ``mongo`` Shell
-  equivalent to ``printjson()``
-
-- ``printjson()`` to print with :term:`JSON` formatting and equivalent
-  to ``print(tojson(<obj>))``
-
-For more information and examples on cursor handling in the
-:binary:`~bin.mongo` shell, see :doc:`/tutorial/iterate-a-cursor`. See also
-:ref:`mongo-shell-help-cursor` for list of cursor help in the
-:binary:`~bin.mongo` shell.
-
-Multi-line Operations in the ``mongo`` Shell
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-If you end a line with an open parenthesis (``'('``), an open brace
-(``'{'``), or an open bracket (``'['``), then the subsequent lines start
-with ellipsis (``"..."``) until you enter the corresponding closing
-parenthesis (``')'``), the closing brace (``'}'``) or the closing
-bracket (``']'``). The :binary:`~bin.mongo` shell waits for the closing
-parenthesis, closing brace, or the closing bracket before evaluating
-the code, as in the following example:
-
-.. code-block:: javascript
-
-   > if ( x > 0 ) {
-   ... count++;
-   ... print (x);
-   ... }
-
-You can exit the line continuation mode if you enter two blank
-lines, as in the following example:
-
-.. code-block:: javascript
-
-   > if (x > 0
-   ...
-   ...
-   >
-
-Tab Completion and Other Keyboard Shortcuts
--------------------------------------------
-
-The :binary:`~bin.mongo` shell supports keyboard shortcuts. For example,
-
-- Use the up/down arrow keys to scroll through command history. See
-  :ref:`.dbshell <mongo-dbshell-file>` documentation for more
-  information on the ``.dbshell`` file.
-
-- Use ``<Tab>`` to autocomplete or to list the completion
-  possibilities, as in the following example which uses ``<Tab>`` to
-  complete the method name starting with the letter ``'c'``:
-
-  .. code-block:: javascript
-
-     db.myCollection.c<Tab>
-
-  Because there are many collection methods starting with the letter
-  ``'c'``, the ``<Tab>`` will list the various methods that start with
-  ``'c'``.
-
-For a full list of the shortcuts, see :ref:`Shell Keyboard Shortcuts
-<mongo-keyboard-shortcuts>`
-
-
-``.mongorc.js`` File
---------------------
-
-When starting, :binary:`~bin.mongo` checks the user's :envvar:`HOME`
-directory for a JavaScript file named :ref:`.mongorc.js
-<mongo-mongorc-file>`. If found, :binary:`~bin.mongo` interprets the
-content of :file:`.mongorc.js` before displaying the prompt for the
-first time. If you use the shell to evaluate a JavaScript file or
-expression, either by using the :option:`--eval <mongo --eval>`
-option on the command line or by specifying :ref:`a .js file to
-mongo <mongo-shell-file>`, :binary:`~bin.mongo` will read the
-``.mongorc.js`` file *after* the JavaScript has finished processing.
-You can prevent ``.mongorc.js`` from being loaded by using the
-:option:`--norc <mongo --norc>` option.
-
-.. _mongo-shell-exit:
-
-Exit the Shell
---------------
-
-To exit the shell, type ``quit()`` or use the ``<Ctrl-C>`` shortcut.
```

Comment:
Is there an equivalent for these sections for mongosh?

Reply:
That would be here: [Disconnect from a Deployment](https://docs.mongodb.com/mongodb-shell/connect/#disconnect-from-a-deployment)

It's possible it should instead go here in the [mongosh Usage](https://docs.mongodb.com/mongodb-shell/run-commands/) page

Reply:
We also have [DOCSP-15488](https://jira.mongodb.org/browse/DOCSP-15488) to audit the legacy shell material and move anything that's missing to the mongosh docs.

Reply:
I was thinking about the preceding sections as well, but I think DOCSP-15488 is the right thing then :+1:

Reply:
Yes, this is just one of several expected tickets.